style.css

```diff
@@ -115,14 +117,70 @@ header > i {
   display: none;
 }
 
+main {
+  background: var(--dark-grey);
+  padding: 60px;
+  border-radius: 23px;
+  margin: 25px auto 0;
+  max-width: 1280px;
+}
+
+#banner {
+  background: url("./assets/images/banner-bg.jpeg") center center/cover no-repeat;
+  min-height: 380px;
+  border-radius: 23px;
+  padding: 80px 60px;
+}
+
+.banner__text {
+  width: 500px;
+}
+
+.banner__text h6 {
+  color: var(--off-white);
+  font-weight: 400;
+  font-size: 20px;
+  margin: 0 0 25px 0;
+}
+
+.banner__text h4 {
+  font-size: clamp(1.5rem, 9.5vw, 2.5rem);
+  line-height: clamp(2.5rem, 14vw , 3.5rem);
+  /* line-height: 55px; */
+  text-transform: uppercase;
+  color: var(--off-white);
+  margin: 0 0 25px 0;
+}
+
+.banner__text h4 span {
+  color: var(--primary-color);
+}
+
+.primary-btn {
+  background: var(--primary-color);
+  color: var(--off-white);
+  border: none;
+  outline: none;
+  padding: 12px 27px;
+  border-radius: 25px;
+  cursor: pointer;
+  font-size: .9rem;
```

Comment:
Make sure you are using REM unit font sizes correctly otherwise this can create issues on responsiveness